src/components/NavItems.js

```diff
@@ -22,9 +22,9 @@ export function NavItems() {
   ]
 
   //adds favorite to nav if user is logged in
-  if (isLoggedIn) {
-    navItems.push({ navLinks: "Favorites", menuLinks: [] })
-  }
+  // if (isLoggedIn) {
+  //   navItems.push({ navLinks: "Favorites", menuLinks: [] })
+  // }
 
```

Comment:
but I think we need to uncomment this. I think we still need the favorties icon in the nav if user is logged in.